src/tests.rs

```diff
@@ -231,20 +231,6 @@ export function fooFn(a: number) {
         "isAsync": false,
         "isGenerator": false
       },
-    },
-    {
-      "kind": "import",
-      "name": "buzz",
-      "location": {
-        "filename": "file:///test.ts",
-        "line": 3,
-        "col": 0
-      },
-      "jsDoc": null,
-      "importDef": {
-        "src": "file:///reexport.ts",
-        "imported": "fizz",
-      }
```

Comment:
Why are those removed from the test?

Reply:
I had originally intended to entirely replace the existing import handling with handling them as reexports, but I just realised they're still needed for linking to types which aren't reexported by the module but are still included in the interface. I'll put it back.